KomaMRICore/src/simulation/Bloch/BlochSimulationMethod.jl

```diff
@@ -19,7 +19,6 @@ function initialize_spins_state(
     Mxy = zeros(T, Nspins)
     Mz = obj.ρ
     Xt = Mag{T}(Mxy, Mz)
-    sort_motions!(obj.motion)
```

Comment:
Yes, since it was only necessary for SimpleMotion and it creates a copy of the motion object. This is assumable for SimpleMotion but not for Arbitrary. When we address https://github.com/JuliaHealth/KomaMRI.jl/issues/410, maybe we should come back to sorting motions inside `initialize_spins_state`, but for now, it is not necessary